app/src/main/java/com/mayokunadeniyi/instantweather/data/model/SearchResult.kt

```diff
@@ -5,7 +5,8 @@ package com.mayokunadeniyi.instantweather.data.model
  */
 
 data class SearchResult(
+    val uId: Int,
     val name: String,
-    val country: String,
-    val subcountry: String
+    val country: String
+   // val subcountry: String
```

Comment:
While we still have to optimise the way we implement search, It would be best to fully qualify the location the user wants to search by letting them see the name of the location, the sub-country and the country it is in. We should leave the sub-country.